Helper/Search/Filter.php

```diff
@@ -179,16 +186,26 @@ public function getChoices(): array
         return $this->choices;
     }
 
+    public function isNested(): bool
+    {
+        return null !== $this->nestedPath;
+    }
+
+    public function getNestedPath(): ?string
```

Comment:
phpdame: ?string not compliant with var definition :)

Reply:
phpDAVMA: check constructor nestedPath can be null, so var definition is wrong